source/target/freescale/k64f/target.c

```diff
@@ -26,11 +26,13 @@
 
 // target information
 target_cfg_t target_device = {
-    .sector_size    = 4096,
-    .sector_cnt     = (MB(1) / 4096),
-    .flash_start    = 0,
-    .flash_end      = MB(1),
-    .ram_start      = 0x1FFF0000,
-    .ram_end        = 0x20030000,
-    .flash_algo     = (program_target_t *) &flash,
+    .sector_size        = 0x1000,
+    .sector_cnt         = (0x100000 / 0x1000),
+    .flash_start        = 0x00000000,
+    .flash_end          = 0x00100000,
+    .ram_start          = 0x20000000,
```

Comment:
I think there is a problem with RAM bank detection here.

Reply:
```
<device Dname="MK64FN1M0xxx12">
        <processor Dfpu="1" Dmpu="0" Dendian="Little-endian" Dclock="120000000"/>
        <compile header="Device/Include/MK64F12.h" define="MK64FN1M0xxx12"/>
        <debug svd="SVD/MK64F12.svd"/>
        <memory id="IROM1"                  start="0x00000000" size="0x00100000" startup="1" default="1"/>
        <memory id="IRAM1"                  start="0x20000000" size="0x00030000" init   ="0" default="1"/>
        <memory id="IRAM2"                  start="0x1FFF0000" size="0x00010000" init   ="0" default="0"/>
        <algorithm name="Flash/MK_P1M0.FLM" start="0x00000000" size="0x00100000"             default="1"/>

        <book name="Documents/K64P144M120SF5RM.pdf" title="MK64 120MHz Reference Manual"/>
        <book name="Documents/K64P144M120SF5.pdf"   title="MK64 120MHz Data Sheet"/>
      </device>
```

This is K64F entry in the PDSC file